generator/vulkan/render.zig

```diff
@@ -1035,13 +1035,80 @@ fn Renderer(comptime WriterType: type) type {
                 \\    const Info = struct {
                 \\        name: [:0]const u8,
                 \\        version: u32,
+                \\        instance_functions: InstanceCommandFlags,
+                \\        device_functions: DeviceCommandFlags,
                 \\    };
             );
+            // The commands in an extension are not pre-sorted based on if they are instance or device functions.
+            var instance_commands = std.ArrayList([]const u8).init(self.allocator);
+            defer instance_commands.deinit();
+            var device_commands = std.ArrayList([]const u8).init(self.allocator);
+            defer device_commands.deinit();
             for (self.registry.extensions) |ext| {
                 try self.writer.writeAll("pub const ");
                 try self.writeIdentifierWithCase(.snake, trimVkNamespace(ext.name));
                 try self.writer.writeAll("= Info {\n");
                 try self.writer.print(".name = \"{s}\", .version = {},", .{ ext.name, ext.version });
+                // collect extension functions
+                for(ext.requires) |require| {
```

Comment:
Can you please run zig fmt on the code base? I'll make sure to set up a check for this in CI.